src/theme/index.css

```diff
@@ -372,6 +393,15 @@ body {
   color: inherit !important;
 }
 
+@keyframes fjzahul {
+  from {
+    background-color: #1a1a1a;
+  }
+  to {
+    background-color: #111;
+  }
+}
+
```

Comment:
1. Please use `var` from the `:root` for colors
2. Please leave a comment above the `@keyframes` what this does